test/destroyed_at_test.rb

```diff
@@ -260,3 +260,11 @@
   end
 end
 
+describe 'destroying on object should call after_commit callback' do
+  it 'calls after_commit callback on: :destroy' do
+    comment = Comment.create
+    comment.destroy
+
+    comment.after_committed.must_equal true
```

Comment:
This spec does not demonstrate that destroying objects fires the after_commit callback. Indeed, it will still pass if line 266 is removed altogether. This is because the `#create` also runs the after_commit callbacks, so the flag is already set before `#destroy` is even run.